docs/fsharp/get-started/get-started-vscode.md

```diff
@@ -1,83 +1,48 @@
-
-If you prefer not to install Visual Studio, use the following instructions:
-
-1. Install [.NET Framework 4.5 or higher](https://www.microsoft.com/en-US/download/details.aspx?id=30653) if you're running Windows 7.  If you're using Windows 8 or higher, you do not need to do this.
-
-2. Install the Windows SDK for your OS:
-
-    * [Windows 10 SDK](https://dev.windows.com/en-US/downloads/windows-10-sdk)
-    * [Windows 8.1 SDK](https://developer.microsoft.com/windows/downloads/sdk-archive)
-    * [Windows 8 SDK](https://developer.microsoft.com/windows/downloads/sdk-archive)
-    * [Windows 7 SDK](https://www.microsoft.com/download/details.aspx?id=8279)
-
-3. Install the [Microsoft Build Tools 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48159).  You may also need to install [Microsoft Build Tools 2013](https://www.microsoft.com/en-us/download/details.aspx?id=40760).
-
-4. Install the [Visual F# Tools](https://www.microsoft.com/en-us/download/details.aspx?id=48179).
-
-On 64-bit Windows, the compiler and tools are located here:
-
-```
-C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsc.exe
-C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsi.exe
-C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsiAnyCpu.exe
-```
-
-On 32-bit Windows, the compiler tools are located here:
-
-```
-C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsc.exe
-C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsi.exe
-C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsiAnyCpu.exe
-```
-
-Ionide automatically detects the compiler and tools, but if it doesn't for some reason (for example, the Visual F# Tools were installed to a different directory), you can manually add the containing folder (`...\Microsoft SDKs\F#\4.0`) to your PATH.
-
-### macOS
-
-On macOS, Ionide uses [Mono](https://www.mono-project.com).  The easiest way to install Mono on macOS is via Homebrew.  Simply type the following into your terminal:
+Ionide uses [Mono](http://www.mono-project.com).  The easiest way to install Mono on macOS is via Homebrew.  Simply type the following into your terminal:
 
 ```
 brew install mono
 ```
 
-### Linux
+### [Linux](#tab/linux)
 
-On Linux, Ionide also uses [Mono](https://www.mono-project.com).  If you're on Debian or Ubuntu, you can use the following:
+On Linux, Ionide also uses [Mono](http://www.mono-project.com).  If you're on Debian or Ubuntu, you can use the following:
 
 ```
 sudo apt-get update
 sudo apt-get install mono-complete fsharp
 ```
 
+### [Windows](#tab/windows)
+
+If you're on Windows, you must [install Visual Studio with F# support](get-started-visual-studio.md#installing-f).  This installs all the necessary components to write, compile, and execute F# code.
```

Comment:
Since installing VS is not the only option, I wouldn't use the word "must" here, even if you're not going to mention the other option.

Also, I find it weird that to use F# in one editor (VS Code), the easiest option is to first install another editor (VS). But I don't think you can fix that by changing what the docs say. 😄

Reply:
Until we complete F# support for FSI and Type Providers on .NET Core, Ionide must take a dependency on either Mono or .NET Framework and their build tools. This means that you need to manually install the .NET Framework (win7 or higher), F# Compiler SDK,  MSBuild, and optionally set your PATHs appropriately if Ionide cannot detect them (reported before so it should be mentioned).

Having watched people struggle mightily with this in workshops, even with this doc, I've had them "just install VS" to make it easier and they were appreciative 😢. The removal here is based on this.

And actually, now that you've hit on this, this is not finished yet, since the .NET Core SDK is required for Ionide as well. I'll need to update that.